.pycheckers

```diff
@@ -0,0 +1,2 @@
+[DEFAULT]
+pylint_rcfile = .pylintrc
```

Comment:
newline

Reply:
This file simply shouldn't be there -- it's a configuration file used by emacs, and should be ignored. I'll fix that, too.